src/main/java/com/ninjaguild/dragoneggdrop/commands/DragonSpawnCmd.java

```diff
@@ -0,0 +1,80 @@
+package com.ninjaguild.dragoneggdrop.commands;
+
+import com.ninjaguild.dragoneggdrop.DragonEggDrop;
+import com.ninjaguild.dragoneggdrop.management.DEDManager;
+import com.ninjaguild.dragoneggdrop.management.EndWorldWrapper;
+import org.bukkit.ChatColor;
+import org.bukkit.World;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.TabCompleter;
+import org.bukkit.entity.EnderCrystal;
+import org.bukkit.entity.EnderDragon;
+import org.bukkit.entity.Player;
+
+import java.util.Collections;
+import java.util.List;
+
+public class DragonSpawnCmd implements CommandExecutor, TabCompleter
+{
```

Comment:
Try to stay true to the programming style of this repository (K&R bracket indentation, tabs vs spaces, etc.)